packages/core/src/config/types.ts

```diff
@@ -268,7 +268,7 @@ interface AbstractFoundryTransaction {
     type: string | null
     from: string | null
     gas: string | null
-    data: string | null
+    input: string | null
```

Comment:
The foundry devs changed the `data` field to `input` in the broadcast file. This is a prime example of why we have to be thoughtful about what parts of Foundry we rely on and why it's important that we use Foundry features in their intended ways. If we still relied on the broadcast file, this update would have randomly broken our entire plugin.